### DISCUSSION TITLE
Coil sensitivity map

### DISCUSSION THREAD

**Discussion by XiongchaoChen** [facebookresearch/fastMRI#254] [General]
Hi, I want to politely ask how to genrate the coil sensitivity maps for multi-coil brain MRI (16 coils, for example). Is the coil sensitivity map the same for each case with the same coil counts? Thank you!

**Comment by mmuckley** [comment]
Hello @XiongchaoChen, in this repository we don't have a dedicated algorithm for doing it. It is implicitly part of the End-to-End Variational Network (see code [here](https://github.com/facebookresearch/fastMRI/blob/684fb128e4dcb4076d4a2870d28c19a672949f51/fastmri/models/varnet.py#L129-L223)).

For dedicated methods just for sensitivity coils, a good starting point is probably the [ESPIRiT paper](https://onlinelibrary.wiley.com/doi/full/10.1002/mrm.24751), which is implemented in the [BART repository](https://github.com/mrirecon/bart) and has been used by many researchers.
